app/main.py

```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(command: str) -> None:
+    func, file_1, directory = command.split()
```

Comment:
bad name for variables, use more descriptive names for naming, f.e. func stores command name, so u can name it like "command" or smth else

Reply:
Fixed